File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    output = ""
    if number % 3 == 0:
        output += "Pling"
    if number % 5 == 0:
        output += "Plang"
    if number % 7 == 0:
        output += "Plong"
    elif number % 3 != 0 and number % 5 != 0 and number % 7 != 0:
        return str(number)
    return output

File: solutions/cpp/raindrops/2/raindrops.cpp
#include "raindrops.h"
#include <string>

namespace raindrops {

std::string convert(short n) {
  if (n % 3 != 0 && n % 5 != 0 && n % 7 != 0)
    return std::to_string(n);
  std::string sound = "";
  if (n % 3 == 0)
    sound += "Pling";
  if (n % 5 == 0)
    sound += "Plang";
  if (n % 7 == 0)
    sound += "Plong";
  return sound;
}

} // namespace raindrops
